src/NewComment.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import { postComment } from "./api";
+import { useParams } from "react-router-dom";
+import Comments from "./Comments";
+import "./NewComment.css";
+
+export default function NewComment() {
+	const { article_id } = useParams();
+	const [comment, setComment] = useState("");
+	const [postedComment, setPostedComment] = useState("");
+	function handleChange(e) {
+		setComment(e.target.value);
+	}
+	function handleSubmit(e) {
+		e.preventDefault();
+		postComment(article_id, comment).then((response) => {
+			if (response.message === "Network Error") {
```

Comment:
This is also a highly specific error that won't get thrown by all browsers (e.g. firefox, there's no way to test this as you can't toggle the network connection off) so it isn't something we need to handle! :)